lib/protocol/Writer.js

```diff
@@ -437,7 +437,11 @@ Writer.prototype.writeCharacters = function writeCharacters(value, encoding) {
   var type = encoding === 'ascii' ? TypeCode.STRING : TypeCode.NSTRING;
   var length = Buffer.byteLength(value, encoding);
   var buffer;
-  if (length <= 245) {
+  if (length === 0 ) {
```

Comment:
We shouldn't make workarounds in the driver in order to support buggy versions of NodeJS.